test/e2e/k8s/operator.go

```diff
@@ -33,7 +34,7 @@ import (
 	"github.com/mongodb/mongodb-atlas-kubernetes/pkg/controller/watch"
 )
 
-func RunOperator(initCfg *Config) (manager.Manager, error) {
+func BuildManager(initCfg *Config) (manager.Manager, error) {
```

Comment:
Oh I see, we are renaming here. But still, I do not see the difference that explains the semantic change.

Are we building stuff that is not being run yet, or are we also launching it?
I guess we are only building and the semantic of the name before was misleading?

Reply:
this function was only creating the struct, so it is just building, not running.